src/Nowruz/modules/orgSettings/components/team/index.tsx

```diff
@@ -0,0 +1,110 @@
+                    <p className='text-sm font-normal text-Gray-light-mode-600 pt-1'>
+                    Manage your team members and their account permissions here.  
+                    </p>     
+                </div>
+            </div>
+
+            <div className="grid grid-cols-5 gap-4 pt-6">
+                <div>
+                    <h6 className="text-sm font-semibold text-Gray-light-mode-700">Invite team members</h6>
+                    <p className="text-sm font-normal text-Gray-light-mode-600">
+                    Invite member to post or manage your organization
+                    </p>
+                </div>
+                <div className="col-span-3">
+                    <SearchDropdown
+                    placeholder="Please select"
+                    icon="image-user"
+                    options={memberList}
+                    isSearchable
+                    onChange={(option) => onSearchMember(option.value)}
+                    />
+                </div>
+                <div>
+                    <SearchDropdown
+                        placeholder="Please select"
+                        options={role}
+                        isSearchable
+                    />
+                </div>
+            </div>
+            <div className="grid grid-cols-5 gap-4 align-bottom mt-4">
+                <div>&nbsp;</div>
+                <div className="col-span-3">
+                    + Add another 
+                </div>
+                <div className="">
+                    <Button color="primary" size="medium" 
+                    startIcon={<Icon name='mail-01'  fontSize={20} color={'#fff'}/>} onClick={()=> {onAddMember(userId)}}>
+                        Send invites
+                    </Button>
+                </div>
+            </div>
+
+            <div className={css.borderSection}></div>
+            <div className="grid grid-cols-5 gap-4 pt-6">
+                <div className="pt-6">
+                    <h6 className="text-sm font-semibold text-Gray-light-mode-700">Invite team members</h6>
+                    <p  className="text-sm font-normal text-Gray-light-mode-600">
+                    Invite member to post or manage your organization
+                    </p>
+                </div>
+                <div className="col-span-4">
+                    <div className={css.tableCereditList}>
+                        <table>
+                            <thead className="border border-Gray-light-mode-200">
+                                <tr className="text-xs font-medium text-Gray-light-mode-600">
+                                    <th>Name</th>
+                                    <th>Role</th>
+                                    <th>&nbsp;</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {members?.items.map((i)=> (
```

Comment:
`i` is not clear for other developers and generally considered as `index`.
please use `member` instead to represent each of the members.

Reply:
Same here